src/element.js

```diff
@@ -36,7 +36,21 @@ function findElement(sendSessionCommand, strategy, selector) {
             getText: getText.bind(null, sendElementCommand),
             getCss: getCss.bind(null, sendElementCommand)
         };
-    });
+    }).catch(() => undefined);
+}
+
+async function findElement(sendSessionCommand, strategy, selector, maxWaitTime = 0) {
```

Comment:
I don't think we should seperate the 2 functionality, because it would just produce code like
`if elementPresent then findElement`. Right now `findElement` without `maxWaitTime` is just an instant query if the element is present. With `maxWaitTime` it waits for it to appear. One way to improve might be using exponential backoff instead of the given 20ms, but that might be an overkill.